Spectrum_Peak_Visualization.py

```diff
@@ -0,0 +1,108 @@
+import matplotlib.pyplot as plt
+import SPEFile
+from ROI_Maker import ROI_Maker
+
+def plot_spectrum(spectrum, title= None, energy_range= None):
+    """
+    plot_spectrum plots the spectra only.
+    """
+    zero_offset = spectrum.energy_cal[0]
+    energy_per_channel = spectrum.energy_cal[1]
+    energy_axis = zero_offset + energy_per_channel*spectrum.channel
+    counts = spectrum.data
+
+    plt.semilogy(energy_axis, counts, drawstyle='steps-mid')
+    plt.xlabel('Energy (keV)')
+    plt.ylabel('Counts')
+    
+    if title != None:      
+        plt.title(title)
+
+    if energy_range != None:
+        plt.xlim([energy_range[0], energy_range[1]])
+
+
+def plot_peaks(spectrum, title= None, energy_range= None, subregion='both'):
+    """
+    plot_peaks plots spectra with peaks highlighted.
+    """
+    if title != None:
+        plt.title(title)
+    
+    counts = spectrum.data    
+    zero_offset = spectrum.energy_cal[0]
+    energy_per_channel = spectrum.energy_cal[1]
+    energy_axis = zero_offset + energy_per_channel*spectrum.channel
+    plt.plot(energy_axis, counts)
+    
+    if subregion == 'both':
+    
+        ROI_info = ROI_Maker(spectrum, subregion)
+        center_peak_region = ROI_info[2]
+        left_peak_region = ROI_info[3]
+        right_peak_region = ROI_info[4]   
+        
+        for i in range(int(len(center_peak_region)/2)):
+            plt.fill_between(energy_axis, counts, 
+            where=(energy_axis >= energy_axis[center_peak_region[2*i]]) &
+            (energy_axis <= energy_axis[center_peak_region[2*i+1]]), 
+            facecolor='r')
```

Comment:
This is hard to read because the whole statement is on the same indentation level. It's also a long statement.
See [PEP 8](https://www.python.org/dev/peps/pep-0008/#code-lay-out) on indentation, and Atom's linter should tell you also (I know it's a little confusing at first).

Good indentation would be either:
```
plt.fill_between(
    energy_axis, counts,
    where=...
```
or
```
plt.fill_between(energy_axis, counts,
                 where=...
```
but if you do line continuation inside of the `where` clause then you add another indent level.

But, it might be even more readable if you define the `where` conditions separately into a variable, so that the `plt.fill_between` line is shorter. I'll leave it up to you whether you want to do that.

Also in the other `fill_between` statements in this file.